report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/trial/EligibleTrialTableGenerator.kt

```diff
@@ -117,11 +136,17 @@ class EligibleTrialsGenerator(
 
         fun forClosedCohorts(
             cohorts: List<InterpretedCohort>, requestingSource: TrialSource?, contentWidth: Float
-        ): EligibleTrialsGenerator {
+        ): EligibleTrialTableGenerator {
             val unavailableAndEligible = cohorts.filter { trial: InterpretedCohort -> trial.isPotentiallyEligible && !trial.isOpen }
             val title =
                 "Trials and cohorts that are potentially eligible, but are closed (${unavailableAndEligible.size})"
-            return create(unavailableAndEligible, emptySet(), requestingSource, null, title, contentWidth, null)
+            return create(unavailableAndEligible, emptySet(), requestingSource, null, title, contentWidth, null, false)
+        }
+
+        fun forFilteredTrials(trials: Set<ExternalTrialSummary>, homeCountry: Country, contentWidth: Float
+        ): EligibleTrialTableGenerator {
```

Comment:
Unusual line break; if this doesn't fit on one line it should be:
```suggestion
        fun forFilteredTrials(
            trials: Set<ExternalTrialSummary>, homeCountry: Country, contentWidth: Float
        ): EligibleTrialTableGenerator {
```